outside.html

```diff
@@ -96,14 +102,31 @@
         <sr-comp id="comp"></sr-comp>
 
         <core-pages id="pages" selected="0">
-            <sr-competitor-schedule data-title="Schedule" data-needs-comp></sr-competitor-schedule>
-            <sr-leaderboard data-title="League Leaderboard" data-needs-comp></sr-leaderboard>
-            <sr-scores data-title="League Scores" data-needs-comp></sr-scores>
-            <sr-knockout-diagram data-title="Knockouts" data-needs-comp></sr-knockout-diagram>
+            <sr-competitor-schedule
+                data-title="Schedule"
+                data-website-url-slug="schedule"
+                data-needs-comp
+            ></sr-competitor-schedule>
+            <sr-leaderboard
+                data-title="League Leaderboard"
+                data-website-url-slug="league"
+                data-needs-comp
+            ></sr-leaderboard>
+            <sr-scores
+                data-title="League Scores"
+                data-website-url-slug="league"
+                data-needs-comp
+            ></sr-scores>
+            <sr-knockout-diagram
+                data-title="Knockouts"
+                data-website-url-slug="knockout"
+                data-needs-comp
+            ></sr-knockout-diagram>
         </core-pages>
 
         <sr-progress id="progress" progress="1">
             <h1 id="progress-title"></h1>
+            <h2 id="website-url"></h2>
```

Comment:
The URL isn't a heading so it shouldn't really be in an `<h2>`, though in this case it doesn't make a lot of difference

Reply:
b9e9bd97094dd1570b1e74590ed00822dce0662d